.gitignore

```diff
@@ -5,6 +5,7 @@
 hfuzz_target
 hfuzz_workspace
 **/Cargo.lock
+deployments/monitoring/grafana/provisioning/dashboards/*.json
```

Comment:
https://github.com/paritytech/parity-bridges-common/pull/831#discussion_r596087706 . The files are physically copied to the monitoring/grafana/provisioning/dashboards/ folder

Reply:
Why do we mount the entire folder though? Can we only add a sub-folder to make sure that there are no local modifications done by the container? Or maybe there is a way to mount read-only?

Reply:
> Why do we mount the entire folder though? Can we only add a sub-folder to make sure that there are no local modifications done by the container?

I don't get your questions. If we don't mount 'the entire folder' (/etc/grafana/provisioning/dashboards), then where we're going to mount subfolders? And where from we need to read dashboards from?

> Or maybe there is a way to mount read-only?

Readonly mount fails, because we then try to populate this folder with dashboards.